assets/js/theme/product/image-gallery.js

```diff
@@ -74,6 +76,18 @@ export default class ImageGallery {
             alt: this.currentImage.mainImageAlt,
             title: this.currentImage.mainImageAlt,
         });
+
+        if (isBrowserIE) {
+            const fallbackStylesIE = {
+                'background-image': `url(${this.currentImage.mainImageUrl}&ampimbypass=on)`,
+                'background-position': 'center',
+                'background-repeat': 'no-repeat',
+                'background-origin': 'content-box',
+                'background-size': 'contain',
+            };
+
+            this.$mainImageNested.css(fallbackStylesIE);
```

Comment:
Could we use css class instead of js obj?

Reply:
good point, but we need to change bg-image url depending on chosen image in gallery plus these styles are inline and bad suited for rewriting via css class.